mda_openbabel_converter/OpenBabelParser.py

```diff
@@ -46,9 +86,84 @@
 
 class OpenBabelParser(TopologyReaderBase):
     """
+    For OpenBabel structure
+
     Inherits from TopologyReaderBase and converts an OpenBabel OBMol to a
     MDAnalysis Topology or adds it to a pre-existing Topology. This parser
     does not work in the reverse direction.
+
+    For use examples, please see OpenBabel Class documentation
```

Comment:
Could you replace `OpenBabel ` with the `:class:` syntax so it automatically makes a hyperlink in the docs?